terraform/modules/quorum-cluster-region/quorum-cluster-region-backup-event.tf

```diff
@@ -0,0 +1,399 @@
+
+# This is to be placed in quorum-cluster-region
+
+provider "archive" {}
+provider "random" {}
+provider "null" {}
+
+data "local_file" "private_key" {
+  filename = "${var.private_key_path}"
+}
+
+resource "aws_s3_bucket_object" "object" {
+  count = "${signum(lookup(var.maker_node_counts, var.aws_region, 0))}"
+  depends_on = ["data.aws_kms_ciphertext.EncryptSSHKey", "local_file.EncryptedSSHKey"]
+  bucket = "${aws_s3_bucket.quorum_backup.id}"
+  key    = "${var.enc_ssh_key}"
+  source = "${var.enc_ssh_path}-${var.aws_region}"
+}
+
+resource "aws_sns_topic" "BackupEvent" {
```

Comment:
nit: Applies across this whole page but again, please no camel case in resource names. Using it in the variables like you do with `name_prefix` is fine.

Reply:
fixed